Xamarin.Forms.Core/AlertArguments.cs

```diff
@@ -6,13 +6,14 @@ namespace Xamarin.Forms.Internals
 	[EditorBrowsable(EditorBrowsableState.Never)]
 	public class AlertArguments
 	{
-		public AlertArguments(string title, string message, string accept, string cancel)
+		public AlertArguments(string title, string message, string accept, string cancel, IVisual visual = null)
```

Comment:
I am not sure if we want to do something like this here... we may actually want to query the page at the point we show the alert. I still need to see if that is possible.